packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -138,18 +128,25 @@ module.exports = ({ strapi }) => ({
   },
 
   findPage(opts, uid, populate) {
-    const params = { ...opts, populate: getBasePopulate(uid, populate) };
+    const params = { ...opts, populate: getDeepPopulate(uid, populate, { maxLevel: 1 }) };
```

Comment:
We could keep getBasePopulate to make the code more readable and it could call getDeepPopulate